scripts/exrrfs_mpassit.sh

```diff
@@ -13,8 +13,17 @@ if [[ -z "${ENS_INDEX}" ]]; then
 else
   ensindexstr="/mem${ENS_INDEX}"
 fi
-${cpreq} ${DATAROOT}${ensindexstr}/${RUN}_fcst_${cyc}/history.${timestr}.nc .
-${cpreq} ${DATAROOT}${ensindexstr}/${RUN}_fcst_${cyc}/diag.${timestr}.nc .
+
+file_history=${UMBRELLA_DATA}${ensindexstr}/${RUN}_fcst_${cyc}/history.${timestr}.nc
+file_diag=${UMBRELLA_DATA}${ensindexstr}/${RUN}_fcst_${cyc}/diag.${timestr}.nc
+if [[ -s ${file_history} ]] && [[ -s ${file_diag} ]]; then
+  ln -s ${file_history} .
+  ln -s ${file_diag} .
+else
+  echo "cannot find history file"
+  exit 1
```

Comment:
If we should stop mpassit immediately, we can use the NCO utility: `err_exit`

Reply:
I am working on group mpssit and UPP now. I will make such change in next PR so it can be tested.